src/utils/__test__/SolutionsUtils.spec.js

```diff
@@ -95,3 +95,87 @@ describe('addRunTemplatesParametersIdsDict for a minimal or incomplete solution'
     });
   });
 });
+
+describe('fixNotCompatibleValuesInSolution function unit tests', () => {
+  const PARAMETER_LIST = {
+    parameters: [
+      {
+        varType: '%DATASETID%',
+        options: {
+          enableAddRow: false,
+          subType: 'TABLE',
+          columns: [
+            {
+              field: 'name',
+              type: ['nonResizable', 'nonSortable'],
+              defaultValue: 'TKT',
+            },
+            {
+              field: 'age',
+              type: ['int'],
+              minValue: 0,
+              maxValue: 120,
+              acceptsEmptyFields: true,
+            },
+          ],
+        },
+      },
```

Comment:
You're reusing this object several times below: instead of getting the same object every time with `PARAMETER_LIST.parameters[0]`, you should probably declare it in a constant first (e.g. `const TABLE_PARAMETER = { ... }`)

Reply:
DONE